kubemonkey/kubemonkey.go

```diff
@@ -19,11 +19,10 @@ func durationToNextRun(runhour int, loc *time.Location) time.Duration {
 		glog.V(1).Infof("Debug mode detected!")
 		glog.V(1).Infof("Status Update: Generating next schedule in %.0f sec\n", debugDelayDuration.Seconds())
 		return debugDelayDuration
-	} else {
-		nextRun := calendar.NextRuntime(loc, runhour)
-		glog.V(1).Infof("Status Update: Generating next schedule at %s\n", nextRun)
-		return nextRun.Sub(time.Now())
 	}
+	nextRun := calendar.NextRuntime(loc, runhour)
+	glog.V(1).Infof("Status Update: Generating next schedule at %s\n", nextRun)
+	return time.Until(nextRun)
```

Comment:
use go 1.10's time.Until

Reply:
Should add a note on the Contributing page that the project requires go 1.10+